src/__init__.py

```diff
@@ -216,4 +261,11 @@ def main():
     except Exception as err:  # pylint: disable=broad-except
         print(err)
 
-    print(putmoon(dateobj, numlines, '@', notext, lang))
+    try:
+        hemisphere = str(args['hemisphere'])
+    except Exception as err:  # pylint: disable=broad-except
+        print(err)
+
+    print(putmoon(dateobj, numlines, '@', notext, lang, hemisphere))
+
+main()
```

Comment:
```suggestion

```
I should remove the main call here. I added before, and its giving the output twice.